eva/utils/profiler.py

```diff
@@ -42,10 +54,22 @@ def run(self):
         Param: None
 
         Returns: List[Metrics]
-        """
-        # TODO: Implement the actual logic
-        # Use self._classobj's methods to run for various batch sizes
+        """        
+
+        #NOTE This wont work if input_format is -1. Need to implement for that case.
+
+        expected_h = self._classobj.input_format.height
+        expected_w = self._classobj.input_format.width
+        expected_c = self._classobj.input_format.channels
+
+        batch_sizes = [1,10,100,1000] 
```

Comment:
Can we add a few more intervals?

Say: 5. 20, 50, 200, 400, 500, 750?

Reply:
okay